DigitalLearningSolutions.Data/Services/TutorialContentService.cs

```diff
@@ -65,5 +76,33 @@ FROM Tutorials
                      AND CustomisationTutorials.Status = 1;",
             new { candidateId, customisationId, sectionId, tutorialId });
         }
+
+        public TutorialContent? GetTutorialContent(int customisationId, int sectionId, int tutorialId)
+        {
+            return connection.QueryFirstOrDefault<TutorialContent?>(

```

Comment:
Do you need the question mark here? 🙂

Reply:
I think I need one in the return type (because this method returns null if the `customisationId`/`sectionId`/`tutorialId` values are inconsistent with one another, but I don't think I need it in the `QueryFirstOrDefault<...>` generic. I had this in the other method, `GetTutorialInformation`, so I've removed them both in ed148f7 🙂

Reply:
Yes, that's what I meant sorry. I agree it's needed in the return type 🙂